cdn/dovetail-cdn/dovetail-stitch-lambda-filters.yml

```diff
@@ -240,27 +225,20 @@ Resources:
       AlarmName: !Sub "[DovetailStitchLambda][LogErrors] ${EnvironmentType} > 0"
       AlarmActions:
         - Fn::If:
-          - IsProduction
-          - Fn::ImportValue:
-              !Sub "${InfrastructureNotificationsStackName}-OpsErrorMessagesSnsTopicArn"
-          - Fn::ImportValue:
-              !Sub "${InfrastructureNotificationsStackName}-OpsWarnMessagesSnsTopicArn"
+            - IsProduction
+            - Fn::ImportValue: !Sub "${InfrastructureNotificationsStackName}-OpsErrorMessagesSnsTopicArn"
+            - Fn::ImportValue: !Sub "${InfrastructureNotificationsStackName}-OpsWarnMessagesSnsTopicArn"
       InsufficientDataActions:
         - Fn::If:
-          - IsProduction
-          - Fn::ImportValue:
-              !Sub "${InfrastructureNotificationsStackName}-OpsErrorMessagesSnsTopicArn"
-          - Fn::ImportValue:
-              !Sub "${InfrastructureNotificationsStackName}-OpsWarnMessagesSnsTopicArn"
+            - IsProduction
+            - Fn::ImportValue: !Sub "${InfrastructureNotificationsStackName}-OpsErrorMessagesSnsTopicArn"
+            - Fn::ImportValue: !Sub "${InfrastructureNotificationsStackName}-OpsWarnMessagesSnsTopicArn"
       OKActions:
         - Fn::If:
-          - IsProduction
-          - Fn::ImportValue:
-              !Sub "${InfrastructureNotificationsStackName}-OpsErrorMessagesSnsTopicArn"
-          - Fn::ImportValue:
-              !Sub "${InfrastructureNotificationsStackName}-OpsWarnMessagesSnsTopicArn"
-      AlarmDescription:
-        Logged errors on dovetail stitch lambda exceeded 0
+            - IsProduction
```

Comment:
Yeah, a bunch of these indented 4 spaces from the `- Fn::If:`

Reply:
They are being indented relative to the `Fn::If:` key, not the `-` array item, which is actually the way I prefer it, and one of the reasons I use the full function name in these cases over the tag.